elfutils/meta.yaml

```diff
@@ -2,7 +2,7 @@
 
 package:
   name: elfutils
-  version: {{ version }}.memfault0
+  version: {{ version }}.memfault0_libdebuginfod
```

Comment:
I noticed that this version ends up getting sorted as an older version, behind `...memfault0`.
I'll bump it to `.memfault1` to fix that.